README.md

```diff
@@ -1 +1,46 @@
-##Network Service
+# Network Service 
+
+Network layer for creating different set of network requests like GET, POST, PUT, DELETE 
+⚠️ The network layer is under active development.
```

Comment:
nit:
Add an empty line to improve readability.
<img width="896" alt="Screenshot 2023-06-15 at 13 44 32" src="https://github.com/ios-course/simple-network-service/assets/32869814/dfa3007f-6856-4520-880d-af020bc6791f">

```suggestion

Network layer for creating different set of network requests like GET, POST, PUT, DELETE 

⚠️ The network layer is under active development.
```

Reply:
fixed